core/include/traccc/edm/internal_spacepoint.hpp

```diff
@@ -56,9 +56,8 @@ struct internal_spacepoint {
         m_z = sp.global[2];
         m_r = std::sqrt(m_x * m_x + m_y * m_y);
 
-        // Need to fix this part
-        m_varianceR = sp.variance[0];
-        m_varianceZ = sp.variance[1];
+        m_varianceR = 0.;
+        m_varianceZ = 0.;
```

Comment:
Do we have a use for these 2 variables? Shouldn't we remove these as well?

Reply:
I think it's OK to remove them for now. We can revive them anytime if it turns out to be useful later